client/my-sites/customer-home/cards/features/domain-upsell/style.scss

```diff
@@ -28,18 +33,24 @@ body .customer-home__layout .domain-upsell__card {
 
 		.badge {
 			position: absolute;
-			right: 20px;
+			right: 27px;
 			border-radius: 4px;
 			font-size: rem(12px);
 			color: var(--studio-gray-80);
 			font-weight: 500;
+			margin-top: 9px;
+			z-index: 1;
+			@include break-small {
+				right: 63px;
+			}
 		}
 
 		.card {
 			margin-bottom: 5px;
 			box-shadow: none;
 			padding: 8px 16px;
-			height: 36px;
+			min-height: 40px;
+			overflow-x: auto;
```

Comment:
I changed it to `overflow-x: auto` to avoid this little issue on the desktop:

<img width="748" alt="image" src="https://user-images.githubusercontent.com/1044309/214936353-65338911-ea06-4543-bd68-d13a8dd329df.png">

I was wondering if we should instead break the domain once it get too big by adding this to the `.card`:

```
    word-break: break-all;
    padding-right: 80px;
```


This would be the result:


<img width="365" alt="image" src="https://user-images.githubusercontent.com/1044309/214936672-466febfc-c572-4ab1-b6b2-b22f62ed9b5c.png">

The decision is up to you. The current strategy also solves the issue 💪

Reply:
Thank, Paulo! I like the word-break idea. I think that looks good. Let's do that. I implemented that using flex box so it takes into consideration different widths for the "Current" and "Available" badges, for translations for example. 🙇